google/README.txt

```diff
@@ -11,21 +11,18 @@ our development, so are not necessarily generalized or applicable outside GCE.
 General layout is this:
 
 google/
-   cassandra   Cassandra schemas, perhaps these could go to config.
-
-   config      Our *-local.yml overrides and default master configuration.
-               These are installed as "config_templates".
+   codelab     (Ignore) Supporting script for building codelab image
```

Comment:
Hmm.. not sure if that's the objective here, and for that reason I explicitly call out (Ignore) on this dir.